guides/v3.7.0/templates/input-helpers.md

```diff
@@ -63,6 +74,9 @@ helper to create a checkbox by setting its `type`:
 {{input type="checkbox" name="isAdmin" checked=this.isAdmin}}
 ```
 
+Being a special input type, [event names](https://www.emberjs.com/api/ember/release/classes/Component) supported on checkbox
```

Comment:
I'm wondering if it might make sense to update the wording in such that it reflects what these event names do and provide a short example, too:

```md
Checkboxes are a special input type. If you want to dispatch an action on a certain event, you will always need to define the event name in camelcase format:
```

```hbs
  {{input type="checkbox" keyPress=(action "updateName")}}
```

and if it would be useful to move the explanation for this special case below the general explanation of checkbox properties (e.g. right after [this line](https://github.com/ember-learn/guides-source/pull/362/files#diff-7043b25baf8b0ac298ad07a60b27d22aR91)).

What are your thoughts?

Reply:
Agreed, I think that explanation is much better. I'll move the example as well 😄